molecule/common/molecule/1_vm_not_idempotence.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Maybe `*_no_idempotece`?

Reply:
What is better: `not idempotent` or `no idempotence`?